---
layout: page
title: Single Cell
permalink: /Single_Cell/
---

# SINGLE CELL

1. Wang, Yong, et al. Clonal evolution in breast cancer revealed by single nucleus genome sequencing. Nature 512.7513 (2014): 155. [https://www.ncbi.nlm.nih.gov/pubmed/25079324](https://www.ncbi.nlm.nih.gov/pubmed/25079324)

   >Sequencing studies of breast tumour cohorts have identified many prevalent mutations, but provide limited insight into the genomic diversity within tumours. Here we developed a whole-genome and exome single cell sequencing approach called nuc-seq that uses G2/M nuclei to achieve 91% mean coverage breadth. We applied this method to sequence single normal and tumour nuclei from an oestrogen-receptor-positive (ER(+)) breast cancer and a triple-negative ductal carcinoma. In parallel, we performed single nuclei copy number profiling. Our data show that aneuploid rearrangements occurred early in tumour evolution and remained highly stable as the tumour masses clonally expanded. In contrast, point mutations evolved gradually, generating extensive clonal diversity. Using targeted single-molecule sequencing, many of the diverse mutations were shown to occur at low frequencies (<10%) in the tumour mass. Using mathematical modelling we found that the triple-negative tumour cells had an increased mutation rate (13.3×), whereas the ER(+) tumour cells did not. These findings have important implications for the diagnosis, therapeutic treatment and evolution of chemoresistance in breast cancer.

1. Butler, Andrew, et al. Integrating single-cell transcriptomic data across different conditions, technologies, and species. Nature Biotechnology 36.5 (2018): 411. [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6700744/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6700744/)

   >Computational single-cell RNA-seq (scRNA-seq) methods have been successfully applied to experiments representing a single condition, technology, or species to discover and define cellular phenotypes. However, identifying subpopulations of cells that are present across multiple data sets remains challenging. Here, we introduce an analytical strategy for integrating scRNA-seq data sets based on common sources of variation, enabling the identification of shared populations across data sets and downstream comparative analysis. We apply this approach, implemented in our R toolkit Seurat (http://satijalab.org/seurat/), to align scRNA-seq data sets of peripheral blood mononuclear cells under resting and stimulated conditions, hematopoietic progenitors sequenced using two profiling technologies, and pancreatic cell 'atlases' generated from human and mouse islets. In each case, we learn distinct or transitional cell states jointly across data sets, while boosting statistical power through integrated analysis. Our approach facilitates general comparisons of scRNA-seq data sets, potentially deepening our understanding of how distinct cell states respond to perturbation, disease, and evolution.

1. Qiu, Xiaojie, et al. Reversed graph embedding resolves complex single-cell trajectories. Nature Methods 14.10 (2017): 979. [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5764547/](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5764547/)

   >Single-cell trajectories can unveil how gene regulation governs cell fate decisions. However, learning the structure of complex trajectories with multiple branches remains a challenging computational problem. We present Monocle 2, an algorithm that uses reversed graph embedding to describe multiple fate decisions in a fully unsupervised manner. We applied Monocle 2 to two studies of blood development and found that mutations in the genes encoding key lineage transcription factors divert cells to alternative fates.

1. Navin, Nicholas, et al. Tumour evolution inferred by single-cell sequencing. Nature 472.7341 (2011): 90. [https://www.ncbi.nlm.nih.gov/pubmed/21399628](https://www.ncbi.nlm.nih.gov/pubmed/21399628)

   >Genomic analysis provides insights into the role of copy number variation in disease, but most methods are not designed to resolve mixed populations of cells. In tumours, where genetic heterogeneity is common, very important information may be lost that would be useful for reconstructing evolutionary history. Here we show that with flow-sorted nuclei, whole genome amplification and next generation sequencing we can accurately quantify genomic copy number within an individual nucleus. We apply single-nucleus sequencing to investigate tumour population structure and evolution in two human breast cancer cases. Analysis of 100 single cells from a polygenomic tumour revealed three distinct clonal subpopulations that probably represent sequential clonal expansions. Additional analysis of 100 single cells from a monogenomic primary tumour and its liver metastasis indicated that a single clonal expansion formed the primary tumour and seeded the metastasis. In both primary tumours, we also identified an unexpectedly abundant subpopulation of genetically diverse 'pseudodiploid' cells that do not travel to the metastatic site. In contrast to gradual models of tumour progression, our data indicate that tumours grow by punctuated clonal expansions with few persistent intermediates.

1. Macosko EZ, Basu A, Satija R, Nemesh J, Shekhar K, Goldman M, Tirosh I, Bialas AR, Kamitaki N, Martersteck EM, Trombetta JJ, Weitz DA, Sanes JR, Shalek AK, Regev A, McCarroll SA. Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets.Cell. 2015 May 21;161(5):1202-1214. doi: 10.1016/j.cell.2015.05.002. [https://www.cell.com/abstract/S0092-8674(15)00549-8](https://www.cell.com/abstract/S0092-8674(15)00549-8)

      >Cells, the basic units of biological structure and function, vary broadly in type and state. Single-cell genomics can characterize cell identity and function, but limitations of ease and scale have prevented its broad application. Here we describe Drop-seq, a strategy for quickly profiling thousands of individual cells by separating them into nanoliter-sized aqueous droplets, associating a different barcode with each cell’s RNAs, and sequencing them all together. Drop-seq analyzes mRNA transcripts from thousands of individual cells simultaneously while remembering transcripts’ cell of origin. Drop-seq will accelerate biological discovery by enabling routine transcriptional profiling at single-cell resolution.

1. Vallejos CA, Risso D, Scialdone A, Dudoit S, Marioni JC.  Normalizing single-cell RNA sequencing data: challenges and opportunities.  Nat Methods. 2017 Jun;14(6):565-571. doi: 10.1038/nmeth.4292. Epub 2017 May 15. [https://www.nature.com/articles/nmeth.4292](https://www.nature.com/articles/nmeth.4292)

      >Single-cell transcriptomics is becoming an important component of the molecular biologist's toolkit. A critical step when analyzing data generated using this technology is normalization. However, normalization is typically performed using methods developed for bulk RNA sequencing or even microarray data, and the suitability of these methods for single-cell transcriptomics has not been assessed. We here discuss commonly used normalization approaches and illustrate how these can produce misleading results. Finally, we present alternative approaches and provide recommendations for single-cell RNA sequencing users.

1. Adamson B, Norman TM, Jost M, Cho MY, et al. Cell. 2016 Dec 15;167(7):1867-1882.e21. doi: 10.1016/j.cell.2016.11.048A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response. [https://www.ncbi.nlm.nih.gov/pubmed/27984733](https://www.ncbi.nlm.nih.gov/pubmed/27984733)

   >Functional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations, and two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1α. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.

1. Dixit A1, Parnas O2, Li B2, Chen J1, Fulco CP3, Jerby-Arnon L2, Marjanovic ND4, Dionne D2, Burks T2, Raychowdhury R2, Adamson B5, Norman TM5, Lander ES6, Weissman JS7, Friedman N8, Regev A9. Perturb-Seq: Dissecting Molecular Circuits with Scalable Single-Cell RNA Profiling of Pooled Genetic Screens.  Cell. 2016 Dec 15;167(7):1853-1866.e17. doi: 10.1016/j.cell.2016.11.038. [https://www.ncbi.nlm.nih.gov/pubmed/27984732](https://www.ncbi.nlm.nih.gov/pubmed/27984732)

      >Genetic screens help infer gene function in mammalian cells, but it has remained difficult to assay complex phenotypes-such as transcriptional profiles-at scale. Here, we develop Perturb-seq, combining single-cell RNA sequencing (RNA-seq) and clustered regularly interspaced short palindromic repeats (CRISPR)-based perturbations to perform many such assays in a pool. We demonstrate Perturb-seq by analyzing 200,000 cells in immune cells and cell lines, focusing on transcription factors regulating the response of dendritic cells to lipopolysaccharide (LPS). Perturb-seq accurately identifies individual gene targets, gene signatures, and cell states affected by individual perturbations and their genetic interactions. By decomposing many high content measurements into the effects of perturbations, their interactions, and diverse cell metadata, Perturb-seq dramatically increases the scope of pooled genomic assays.

1. B. Wang, J. Zhu, E. Pierson, D. Ramazzotti, S. Batzoglou, Visualization and analysis of single-cell RNA-seq data by kernel-based similarity learning. Nat. Methods. 14, 414–416 (2017). [https://www.ncbi.nlm.nih.gov/pubmed/28263960](https://www.ncbi.nlm.nih.gov/pubmed/28263960)

   >We present single-cell interpretation via multikernel learning (SIMLR), an analytic framework and software which learns a similarity measure from single-cell RNA-seq data in order to perform dimension reduction, clustering and visualization. On seven published data sets, we benchmark SIMLR against state-of-the-art methods. We show that SIMLR is scalable and greatly enhances clustering performance while improving the visualization and interpretability of single-cell sequencing data.

1. V. Y. Kiselev et al., SC3: consensus clustering of single-cell RNA-seq data. Nat. Methods. 14, 483–486 (2017). [https://www.ncbi.nlm.nih.gov/pubmed/28346451](https://www.ncbi.nlm.nih.gov/pubmed/28346451)

   >Single-cell RNA-seq enables the quantitative characterization of cell types based on global transcriptome profiles. We present single-cell consensus clustering (SC3), a user-friendly tool for unsupervised clustering, which achieves high accuracy and robustness by combining multiple clustering solutions through a consensus approach ([http://bioconductor.org/packages/SC3](http://bioconductor.org/packages/SC3)). We demonstrate that SC3 is capable of identifying subclones from the transcriptomes of neoplastic cells collected from patients.

1. L. Haghverdi, A. T. L. Lun, M. D. Morgan, J. C. Marioni, Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors. Nat. Biotechnol. 36, 421–427 (2018). [https://www.nature.com/articles/nbt.4091](https://www.nature.com/articles/nbt.4091)

   >Large-scale single-cell RNA sequencing (scRNA-seq) data sets that are produced in different laboratories and at different times contain batch effects that may compromise the integration and interpretation of the data. Existing scRNA-seq analysis methods incorrectly assume that the composition of cell populations is either known or identical across batches. We present a strategy for batch correction based on the detection of mutual nearest neighbors (MNNs) in the high-dimensional expression space. Our approach does not rely on predefined or equal population compositions across batches; instead, it requires only that a subset of the population be shared between batches. We demonstrate the superiority of our approach compared with existing methods by using both simulated and real scRNA-seq data sets, and using multiple droplet-based scRNA-seq data sets, we demonstrate that our MNN batch-effect-correction method can be scaled to large numbers of cells.
